src/demos/calcite-shell.html

```diff
@@ -14,13 +14,53 @@
   </head>
 
   <body>
+    <calcite-popover placement="vertical" id="calcite-position-floating-panel">
+      <calcite-shell-floating-panel
+        hidden
```

Comment:
Does the popover content need to set `hidden`? I would expect the component to handle this.

Reply:
Nope. will set it on the component.